.github/workflows/publish-docs.yml

```
@@ -15,12 +15,16 @@ on:
 env:
   RUST_TOOLCHAIN_VERSION: "1.82.0"
```

Should this also be bumped?

```suggestion
  RUST_TOOLCHAIN_VERSION: "1.84.1"
```

---

Good spotting! I will update the issue template to include this.